modules/programs/msmtp-accounts.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs }:
+{ config, lib, ... }:
+
+with lib;
+
+{
+  options.msmtp = {
+    enable = mkEnableOption "msmtp as sendmail";
+
+    command = {
+      send = mkOption {
+        readOnly = true;
+        type = types.str;
+        defaultText = "msmtp --account=<account name>";
```

Comment:
I guess most MUAs would expect the `-t` flag (i.e., retrive mailinfo from stdin)